test-suites/_base_reserve_configuration.spec.ts

```diff
@@ -387,4 +411,386 @@ describe("ReserveConfiguration", async () => {
     ).to.be.revertedWith(ProtocolErrors.INVALID_LIQUIDATION_PROTOCOL_FEE);
     expect(await configMock.getLiquidationProtocolFee()).to.be.eq(ZERO);
   });
+
+  it("TC-reserve-configuration-20 Initialize an already initialized reserve. ReserveLogic `init` where xTokenAddress != ZERO_ADDRESS (revert expected)", async () => {
+    const {pool, dai, deployer, configurator} = await loadFixture(
+      testEnvFixture
+    );
+
+    // Impersonate PoolConfigurator
+    await topUpNonPayableWithEther(
+      deployer.signer,
+      [configurator.address],
+      utils.parseEther("1")
+    );
+    await impersonateAccountsHardhat([configurator.address]);
+    const configSigner = await hre.ethers.getSigner(configurator.address);
+
+    const config = await pool.getReserveData(dai.address);
+
+    await expect(
+      pool.connect(configSigner).initReserve(
+        dai.address,
+        config.xTokenAddress, // just need a non-used reserve token
+        config.variableDebtTokenAddress,
+        ZERO_ADDRESS,
+        ZERO_ADDRESS
+      )
+    ).to.be.revertedWith(ProtocolErrors.RESERVE_ALREADY_INITIALIZED);
+  });
+
+  it("TC-reserve-configuration-21 Init reserve with ZERO_ADDRESS as xToken twice, to enter `_addReserveToList()` already added (revert expected)", async () => {
+    /**
+     * To get into this case, we need to init a reserve with `xTokenAddress = address(0)` twice.
+     * `_addReserveToList()` is called from `initReserve`. However, in `initReserve` we run `init` before the `_addReserveToList()`,
+     * and in `init` we are checking if `xTokenAddress == address(0)`, so to bypass that we need this odd init.
+     */
+    const {pool, dai, deployer, configurator} = await loadFixture(
+      testEnvFixture
+    );
+
+    // Impersonate PoolConfigurator
+    await topUpNonPayableWithEther(
+      deployer.signer,
+      [configurator.address],
+      utils.parseEther("1")
+    );
+    await impersonateAccountsHardhat([configurator.address]);
+    const configSigner = await hre.ethers.getSigner(configurator.address);
+
+    const config = await pool.getReserveData(dai.address);
+
+    const poolListBefore = await pool.getReservesList();
+
+    expect(
+      await pool
+        .connect(configSigner)
+        .initReserve(
+          config.xTokenAddress,
+          ZERO_ADDRESS,
+          config.variableDebtTokenAddress,
+          ZERO_ADDRESS,
+          ZERO_ADDRESS
+        )
+    );
```

Comment:
```suggestion
    expect(
      await pool
        .connect(configSigner)
        .initReserve(
          config.xTokenAddress,
          ZERO_ADDRESS,
          config.variableDebtTokenAddress,
          ZERO_ADDRESS,
          ZERO_ADDRESS
        )
    );
```
expect(x); inside It seems that no verification has been done